features/account/customer_account/viewing_orders_history/viewing_payment_status_for_an_order.feature

```diff
@@ -0,0 +1,29 @@
+@customer_account
+Feature: Viewing payment's status on a placed order show page
+    In order to know wheater order has already been paid, or not
```

Comment:
I'm not sure about `if an`, but `wheater` should not be here for sure. 
```suggestion
    In order to know if an order has already been paid, or not
```

Reply:
It's ok IMO, if we change it to `whether an Order` 😄

Reply:
we can have whatever here, but not `wheater` :D

Reply:
🌧 ☁️ 🌦 💨 ? 😂